WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -547,26 +751,30 @@ public void onEventMainThread(UploadService.UploadErrorEvent event) {
 
     @SuppressWarnings("unused")
     public void onEventMainThread(UploadService.UploadMediaSuccessEvent event) {
+        AnalyticsTracker.track(Stat.MY_SITE_ICON_UPLOADED);
         EventBus.getDefault().removeStickyEvent(event);
         SiteModel site = getSelectedSite();
-        if (site != null && event.mediaModelList != null && !event.mediaModelList.isEmpty()) {
-            UploadUtils.onMediaUploadedSnackbarHandler(getActivity(),
-                    getActivity().findViewById(R.id.coordinator), false,
-                    event.mediaModelList, site, event.successMessage);
-        }
-    }
 
+        if (site != null) {
+            if (isMediaUploadInProgress()) {
```

Comment:
Do we need the `showSiteIconProgressBar(false)` statement after the upload succeeds and the upload progress is shown (i.e. `isMediaUploadInProgress()`)?